src/client/components/household-information/DeductibleExpensesSection.jsx

```diff
@@ -58,39 +58,25 @@ class DeductibleExpensesSection extends React.Component {
         {notRequiredMessage}
 
         <p>
-          Report non-reimbursed medical expenses paid by you or your spouse.
-          Include expenses for medical and dental care, drugs, eyeglasses,
-          Medicare, medical insurance premiums and other health care expenses
-          paid by you for dependents and persons for whom you have a legal or
-          moral obligation to support. Do not list expenses if you expect to
-          receive reimbursement from insurance or other sources. Report expenses
-          of last illness and burial expenses, e.g., prepaid burial, paid by the
-          Veteran for spouse or dependent(s).
+          Tell us a bit about your expenses this past calendar year.
         </p>
 
         <div className="input-section">
           <ErrorableTextInput
               errorMessage={getErrorMessage(this.props.data.deductibleMedicalExpenses, message)}
-              label="Total non-reimbursed medical expenses paid by you or your spouse
-                  (e.g., payments for doctors, dentists, medications, Medicare, health
-                  insurance, hospital and nursing home) VA will calculate a deductible
-                  and the net medical expenses you may claim."
+              label="Have you or your spouse paid any non-reimbursable medical expenses this past year?"
```

Comment:
I see what you mean about the way the question is worded. Perhaps we can change it to, "Total amount paid for any non-reimbursable medical expenses this past year". Do you think that works? And then we can change the introduction to say "Tell us a bit about your expenses this past calendar year. Enter information for any expenses that apply to you."